### ISSUE TITLE
Documentation on using Scope within TranslocoService

### ISSUE THREAD

**Issue by Paitum** [ngneat/transloco#242]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[X] Documentation issue or request
[X] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I'm finding it difficult to understand how to work with scopes via the Translation API as documented here: https://netbasal.gitbook.io/transloco/service-api/translation-api

I am specifically looking for a way to get the entire translation mapping for a scope, in the same way that `getTranslation` supports getting the mapping for a non-scope.

I can subscribe to a specific translation keys in a scope via `selectTranslate`, but subscribing to each key doesn't scale.

The component I am developing has translations from the base (e.g. en-US, de-DE), as well as from a scope (e.g. page/en-US, page/de-DE). These translations are being passed to child components, some of which come from an external library and even web components, so they can't be responsible for the translations. Therefore I am looking to use the Observables in the TranslocoService to react to a combination of language changes and TranslationLoadSuccess events, which will update the relevant values depending on the scope.

I'm trying all sorts of things like `getTranslation('page/en-US')` [as is hinted](https://github.com/ngneat/transloco/blob/master/projects/ngneat/transloco/src/lib/transloco.service.ts#L289) at in the source-code.

If this functionality is missing then this is more of a support request, otherwise, please update the documentation to better explain how to get a scope's translation mapping whole.

## Expected behavior

Documentation should provide more clarity on how to use scopes within the relevant functions. For example, does `selectTranslateObject('key')` support `selectTranslateObject('scope/key')`?

`translateObject('key', {}, 'scope/lang')` works but requires a key, can I provide an empty string to get the root translation object?

Why doesn't `getTranslateObject` exist?

## Environment

<pre><code>
Angular version: 9.0.0
</code></pre>

**Comment by Paitum** [comment]
After thinking about this more, it seems I need an Observable that triggers whenever the language changes AND the strings are available. So something like:
```
// emits when a language file has been changed-out and is available
this.translocoService.translationChange$.subscribe({scope} => this.updateTranslations(scope));

updateTranslations(scope: TranslocoScope) {
  if(scope === "page") {
    this.valueForComponent.firstName = translate('user.firstname', {}, scope);
    this.valueForComponent.lastName = translate('user.lastname', {}, scope);
  } else {
    this.valueForComponent.outage = translate('errors.outage', {}, scope);
  }
}
```

or a more direct object-access approach like, with less boiler plate:

```
updateTranslations(scope: TranslocoScope) {
  const translations = this.translocoService.translateObject(scope);
  
  if(scope === "page") {
    this.valueForComponent.firstName = translations.user.firstname;
    this.valueForComponent.lastName = translations.user.lastname;
  } else {
    this.valueForComponent.outage = translations.errors.outage;
  }
}
```

**Comment by Paitum** [comment]
Closing this issue since I have figured out some aspects on my own, and don't want to put someone through the hassle of reading my progression above. I hope to return with specific documentation recommendations in the near future.

I think my problem is I didn't setup and configure Transloco in my code, so when accessing the documentation, I found sections assumed the reader understands more than they might.

This was especially true for the Translation API.
